src/lib.rs

```diff
@@ -78,6 +78,8 @@
 //! - `multiple-pymethods`: Enables the use of multiple [`#[pymethods]`](macro@crate::pymethods)
 //! blocks per [`#[pyclass]`](macro@crate::pyclass). This adds a dependency on the [inventory]
 //! crate, which is not supported on all platforms.
+//! - `pyproto`: Adds the deprecated `#[pyproto]` attribute macro. Enabled by default, and likely
```

Comment:
This should be under the Default feature header, together with the macros feature (whose description isnt quite right anymore 🙃)

Reply:
Thanks, moved 👍